AIPscan/Reporter/report_aips_by_puid.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from flask import render_template, request
+
+from AIPscan.Data import data
+from AIPscan.Reporter import reporter, translate_headers
+from AIPscan.models import File
+
+
+def get_format_string_from_puid(puid):
+    """Return file format and version info for a PUID or None
```

Comment:
See main thread re: intent here. I agree that there's room for sharing code between the reports, but they are intended to communicate different levels of detail about file formats and versions. My intent with this helper was really just to provide a helper so that users looking at a report don't have to go look up what a PUID is externally.